src/sass/_game.scss

```diff
@@ -68,6 +68,18 @@
   animation: blink normal 0.8s infinite ease-in-out; /* Opera */
 }
 
+.time-add {
+  color: #16A085;
+   -moz-transition: all 0.2s ease-in-out;
+  -webkit-transition: all 0.2s ease-in-out;
+  -o-transition: all 0.2s ease-in-out;
+  -ms-transition: all 0.2s ease-in-out;
+  transition: all 0.2as ease-in-out; 
```

Comment:
Refer to https://github.com/twbs/bootstrap-sass/blob/master/assets/stylesheets/bootstrap/mixins/_vendor-prefixes.scss